mycqu/card.py

```diff
@@ -37,12 +39,12 @@ class ParseError(Exception):
     """
 
 
-class FeeAcquisitionFailed(Exception):
+class AcquisitionFailed(Exception):
```

Comment:
仔细一想这个词（`Acquisition`）好像不太常见，而且似乎这个异常和 `CQUWebsiteError` 是重合的。（这个 PR 合了之后我打算重构一下，把异常都放到一个文件里了）

Reply:
如果使用 `CQUWebsiteError`，此处可以先从 `score` 导入 `CQUWebsiteError`，或者就在这个 PR 里面新建一个 `exceptions.py` 将 `CQUWebsiteError` 移进去

Reply:
如果使用 `CQUWebsiteError`，此处可以先从 `score` 导入 `CQUWebsiteError`：

```suggestion
from score import CQUWebsiteError
```

或者就在这个 PR 里面新建一个 `exceptions.py` 将 `CQUWebsiteError` 移进去